.github/workflows/check-pr-base.yml

```diff
@@ -0,0 +1,18 @@
+name: Bors
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
```

Comment:
Recommend adding `branches: master` so this does not run and fail on every single PR to old release branches.